pillar/edx/ansible_vars/xpro.sls

```diff
@@ -11,9 +11,28 @@
   } %}
 {% set heroku_env = heroku_xpro_env_url_mapping['{}'.format(purpose)] %}
 {% set support_email = 'xpro@mit.edu' %}
+{% set THEME_NAME = 'mitxpro-theme' %}
 
 edx:
+  edxapp:
+    THEME_NAME: '{{ THEME_NAME }}'
+    custom_theme:
+      repo: {{ purpose_data.versions.theme_source_repo }}
+      branch: {{ purpose_data.versions.theme }}
```

Comment:
This should all be in the `edx/init.sls` pillar, but doesn't hurt to override it here explicitly.